src/components/TypeLayout/TypeLayout.js

```diff
@@ -56,67 +47,76 @@ const TypeLayout = ({
   </StructuredListWrapper>
 );
 
-const TypeLayoutRow = ({ children, className, ...other }) => (
-  <StructuredListRow
-    className={classnames(`${namespace}__row`, className)}
+const TypeLayoutBody = ({ children, className, ...other }) => (
+  <StructuredListBody
+    className={classnames(`${namespace}__body`, className)}
     {...other}
   >
     {children}
-  </StructuredListRow>
+  </StructuredListBody>
 );
 
-const TypeLayoutBody = ({ children, className, ...other }) => (
-  <StructuredListBody
-    className={classnames(`${namespace}__body`, className)}
+const TypeLayoutCell = ({ children, className, ...other }) => (
+  <StructuredListCell
+    className={classnames(`${namespace}__cell`, className)}
     {...other}
   >
     {children}
-  </StructuredListBody>
+  </StructuredListCell>
 );
 
-const defaultProps = {
-  className: '',
-  children: null,
-};
+const TypeLayoutRow = ({ children, className, ...other }) => (
+  <StructuredListRow
+    className={classnames(`${namespace}__row`, className)}
+    {...other}
+  >
+    {children}
+  </StructuredListRow>
+);
 
 const propTypes = {
-  /** @type {string} Extra classes. */
-  className: string,
-
-  /** @type {ReactNode} Component children. */
+  /** Provide the contents of the node */
   children: node,
+
+  /** Provide an optional class to be applied to the containing node */
+  className: string,
 };
 
-TypeLayout.defaultProps = {
-  border: false,
-  bordered: null,
-  className: '',
+const defaultProps = {
```

Comment:
Why is this being abstracted here? I think it's only used in the default props below?

To me this makes things harder to scan and comprehend. Would you consider moving this information down where it is used?

Reply:
Good point! I'll move it down